apps/accounts/serializers.py

```
@@ -133,9 +129,45 @@ def update(self, instance, validated_data):
+
+class UpdateProfileSerializer(serializers.Serializer):
+
+    username = serializers.CharField()
+    first_name = serializers.CharField()
+    last_name = serializers.CharField()
+    chat_id = serializers.IntegerField()
+    email = serializers.EmailField()
+    avatar = serializers.ImageField(required=False)
+
+    def validate_username(self, value):
+        user = CustomUser.objects.filter(username=value)
```

Ruff

`CustomUser` may be undefined, or defined from star imports

(F405)